hs_core/migrations/0030_resourcefile_file_folder.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hs_core', '0029_auto_20161123_1858'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='resourcefile',
+            name='file_folder',
+            field=models.CharField(max_length=255, null=True),
```

Comment:
@alvacouch As a result of changing max_length from 255 to 1024, this needs to be changed as well in the migration file. After this is changed, a +1 from me.